README.rst

```diff
@@ -2,33 +2,33 @@
 ResolveLib
 ==========
 
-ResolveLib at the highest level provides a `Resolver` class that includes
+ResolveLib at the highest level provides a ``Resolver`` class that includes
 dependency resolution logic. You give it some things, and a little information
 on how it should interact with them, and it will spit out a resolution result.
 
 
 Intended Usage
 ==============
 
-::
+.. code-block:: python
 
-    import resolvelib
+   from resolvelib import BaseReporter, Resolver
```

Comment:
Why? I like the original qualified imports better than this.

Reply:
I'll revert then since I don't really have a strong opinion on this one.